public/data/job-openings-data.json

```diff
@@ -1160,11 +1160,44 @@
           "count": 2,
           "experience": "2+ years"
         },
-	  {
+        {
           "title": "Full Stack Developer",
           "count": 3,
           "experience": "3+ years"
+        }
+      ]
+    },
+    {
+      "id": 27,
+      "companyName": "QDev Technolab",
+      "location": "Ahmedabad, India",
+      "bio": "We are software development company who believe in quality and excellence. When Quality meets excellence, it will gives us Innovation as output. QDev has been helping peoples across various industries to optimize their business processes through custom software solutions and convert their ideas to implementation. we also consider our team members as our most valuable assets and we are taking care of them .",
+      "companyLogoUrl": "https://drive.google.com/file/d/1zPpzXGnmW6pExVI2RzKJ_7zgSp3wcaPa/view?usp=sharing",
```

Comment:
This URL will not work, Please check other job objects and raise a new PR.
Merging for now.